metadata.go

```diff
@@ -331,6 +331,9 @@ func getKeyspaceMetadata(
 		`,
 		keyspaceName,
 	)
+	// Set a routing key to avoid GetRoutingKey from computing the routing key
+	// TODO use a separate connection (pool) for system keyspace queries.
+	query.RoutingKey([]byte{})
```

Comment:
My token aware connection pool implementation (future PR) actually introduces a HostSelectionPolicy and ConnSelectionPolicy interfaces with a new policy driven connection pool. When we get to that I figure we can hash out any alterations to that.